parser_library/src/processing/instruction_sets/asm_processor.cpp

```diff
@@ -817,7 +818,34 @@ void asm_processor::process_START(rebuilt_statement stmt)
 
     hlasm_ctx.ord_ctx.set_available_location_counter_value(start_section_alignment, offset);
 }
+void asm_processor::process_END(rebuilt_statement stmt)
+{
+    const auto& label = stmt.label_ref();
+
+    if (!check(stmt, hlasm_ctx, checker_, *this))
+        return;
```

Comment:
The two remaining "validation" returns should be probably removed as well.
I started the benchmark, so we'll wait for the results and I think this will be ready for merging.